CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.22.18] - 2022-02-03
+
+### Fixed
+
+- _write_dispatch_to_python_file isn't called each time a task is saved.
```

Comment:
Can we mention the only place where its called?

Reply:
Yes, I just made the change;
"It is now only called in the final save in _run_planned_workflow (in `covalent_dispatcher/_core/__init__.py`)."